packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -98,27 +98,52 @@ class Chat extends Widget<Properties> {
       text,
     };
 
-    // @ts-expect-error
-    this.renderMessage(message, user);
+    this.renderMessage(message);
     // @ts-expect-error
     this._messageSendAction?.({ message, event });
   }
 
+  _oneMessageWasAddedToTheEnd(value: Message[], previousValue: Message[]): boolean {
```

Comment:
I don't understand why we place this logic in chat component. I believe that we can move it inside 'message_list'. Based on the `items` property that are provided to the message list, component will be able to decide whether it needs to re-render completely or just add a new message.

Reply:
Also it allows to avoid use '_setOptionWithoutOptionChange' inside `_renderMessage` method

Reply:
[9cf23699d12868a12dc107ad3dbba413a4520116]